docs/_guide/decorators.md

```diff
@@ -0,0 +1,260 @@
+
+<div class="alert alert-info">
+
+**The decorators proposal**. Decorators are a stage 2 proposal for addition to the ECMAScript standard, which means they're not implemented in browsers yet. Compilers like Babel provide support for proposed features like decorators by compiling them into JavaScript a browser can run.
+
+</div>
+
+### To use decorators with TypeScript
+
+To use decorators with TypeScript, enable the `experimentalDecorators` compiler option.
+
+```json
+"experimentalDecorators": true,
+```
+
+Enabling `emitDecoratorMetadata` is not required and not recommended.
+
+### To use decorators with JavaScript
+
+If you're  JavaScript with Babel, you can enable decorators by adding  the following plugins:
+
+*   [`@babel/plugin-proposal-decorators`](https://babeljs.io/docs/en/babel-plugin-proposal-decorators). 
+*   [`@babel/plugin-proposal-class-properties`](https://babeljs.io/docs/en/babel-plugin-proposal-class-properties)
+
+To enable the plugins, you'd add code like this to your Babel configuration:
+
+```js
+plugins = [
+  '@babel/plugin-proposal-class-properties',
+  ['@babel/proposal-decorators', {decoratorsBeforeExport: true}],
+];
+```
+
+## LitElement decorators
+
+LitElement provides the following decorators:
+
+*   <code>[customElement](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#customelement)</code>. Define a custom element.
+*   <code>[eventOptions](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#eventoptions)</code>. Add event listener options to a listener added in your template.
+*   <code>[property](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#property)</code> and <code>[internalProperty](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#internalproperty)</code>. Define properties.
+*   <code>[query](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#query)</code>, <code>[queryAll](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#queryAll)</code>, and <code>[queryAsync](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#queryAsync). </code>Create a property getter that returns specific elements from your component's render root.
+*   <code>[queryAssignedNodes](https://lit-element.polymer-project.org/api/modules/_lit_element_.html#queryAssignedNodes)</code>. Create a property getter that returns the children assigned to a specific slot. 
+
+
+All of the decorators can be imported directly from the <code>lit-element</code> module.
+
+```js
+import {eventOptions} from 'lit-element';
+```
+
+
+
+## Other document updates
+
+<<<The following sections belong  in the individual chapters.>>>
+
+
+### customElement
+
+<<<Maybe add this to the to-be-written Writing Components Overview>>>
+
+Use the `customElement` decorator on your component class to define a custom element:
+
+```js
+@customElement('my-element)
```

Comment:
```suggestion
@customElement('my-element')
```

Reply:
Oops. This section ("other document updates") was intended as a set of smaller updates for other docs, and I accidentally left it in here when I converted the doc to markdown. Will fix this before I submit those, just wanted to add an FYI why I was skipping these suggestions.